internal/pkg/manifest/validate.go

```diff
@@ -850,16 +850,42 @@ func (a AdvancedCount) Validate() error {
 		}
 	}
 
+	// Validate combinations with cooldown
+	if !a.Cooldown.IsEmpty() {
+		if !a.hasScalingFieldsSet() {
+			return &errAtLeastOneFieldMustBeSpecified{
+				missingFields:    a.validScalingFields(),
+				conditionalField: "cooldown",
+			}
+		}
+		if !a.CPU.IsEmpty() {
+			if !a.CPU.ScalingConfig.IsEmpty() {
+				return &errFieldMutualExclusive{
+					firstField:  "cooldown",
+					secondField: "cpu_percentage/cooldown",
+				}
+			}
+		}
+		if !a.Memory.IsEmpty() {
+			if !a.Memory.ScalingConfig.IsEmpty() {
+				return &errFieldMutualExclusive{
+					firstField:  "cooldown",
+					secondField: "memory_percentage/cooldown",
+				}
+			}
+		}
+	}
+
 	// Validate individual custom autoscaling options.
 	if err := a.QueueScaling.Validate(); err != nil {
 		return fmt.Errorf(`validate "queue_delay": %w`, err)
 	}
-	if a.CPU != nil {
+	if !a.CPU.IsEmpty() {
```

Comment:
we probably don't need this because of https://github.com/aws/copilot-cli/pull/3629/files#diff-5fc13795122fe030799a09ef956a8b66cec3d40bf1a81266323edbf3626b5685R906